### ISSUE TITLE
cue: len of open-ended list should not evaluate to null

### ISSUE THREAD

**Issue by cueckoo** [cue-lang/cue#81]
_Originally opened by @shasderias in https://github.com/cuelang/cue/issues/81_

Following-up from Slack conversation.
(1)
```
args: [...string]
argsLen: len(args)
```
evaluates to
```
args: []
argsLen: null
```
Expected argsLen to be 0 instead, i.e. expected (1) to evaluate to:
```
args: []
argsLen: 0
```
Possibly a non-issue or a separate issue,
(2)
```
args: *[] | [...string]
argsLen: len(args)
```
also evaluates to
```
args: []
argsLen: null
```
In (2), also expected argsLen to be 0.

**Comment by cueckoo** [comment]
_Original reply by @shasderias in https://github.com/cuelang/cue/issues/81#issuecomment-526409393_

Oops, just saw the commit. Closed by f40ecd9012a23cdbbbb3405d0eb2a38caefdca89?

```
C:\dev\cue_playground\len>go get -u cuelang.org/go/cmd/cue

C:\dev\cue_playground\len>type len.cue
args: [...string]
argsLen: len(args)

C:\dev\cue_playground\len>cue eval len.cue
args: []
argsLen: 0

C:\dev\cue_playground\len>type len.cue
args: *[] | [...string]
argsLen: len(args)

C:\dev\cue_playground\len>cue eval len.cue
args: []
argsLen: 0
```

**Comment by cueckoo** [comment]
_Original reply by @mpvl in https://github.com/cuelang/cue/issues/81#issuecomment-526730593_

The documentation is right: it always returns int (which includes an error in CUE)

The bug is fixed as of v0.0.8  (commit f40ecd901).
